### DISCUSSION TITLE
Node Firebase GET

### DISCUSSION THREAD

**Discussion by Knabbler** [GogoVega/node-red-contrib-firebase-realtime-database#28] [Ideas]
Dear Gauthier,
if the path to the Firebase DB doesn't exist, the node won't generate a message and the following processing stucks.
<no data> is an important information I must take care of.

Kind regards

Knabbler

**Comment by GogoVega** [comment]
Hi Knabbler,
Thank you for your message, indeed if there is no data for a path, `DataSnapshot` is `null` and the method `sendMsg` does not send any message.

I fixed this error [here](https://github.com/GogoVega/node-red-contrib-firebase-realtime-database/commit/5742dbc5de367f9ad15043d91d37e123919bbc23) and now, a `null` payload will be sent.

I will soon release v0.4.0 to integrate this fix, add "Pass through msg" option and authentication with Custom Token 😉

**Reply by Knabbler** [reply]
Sounds good, thank you.
